pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -113,9 +113,7 @@ func (vim *virtIOInterfaceManager) hotUnplugVirtioInterface(vmi *v1.VirtualMachi
 }
 
 func interfacesToHotUnplug(vmiSpecInterfaces []v1.Interface, domainSpecInterfaces []api.Interface) []api.Interface {
-	ifaces2remove := netvmispec.FilterInterfacesSpec(vmiSpecInterfaces, func(i v1.Interface) bool {
-		return i.State == v1.InterfaceStateAbsent
-	})
+	ifaces2remove := netvmispec.FilterIfacesForHotUnplug(vmiSpecInterfaces)
```

Comment:
Only "bridge" interfaces can be "absent", so I find this change redundant.

Reply:
It may seem premature but I think it is worth having it to prevent from other interface types being subject to unplug, it may also serve the futuristic SR-IOV hotplug/unplug support.
WDTY?

Reply:
Absent SRIOV interface will have to be unplugged here as well. Let's keep the code generic.

Reply:
SR-IOV interfaces cannot be handled at this place because its backed by hostDevice and not a regular interface in the domain spec.

Reply:
I think you meant for the filtering in virt-handler, if so I removed that commit as we concluded offline, in favor of keeping the implementation generic at virt-handler level.